### PR TITLE
Initial pass at project configuration

### PR DESCRIPTION
**Pull request by nickschuch**
Ported over from previous internal discussions.

### CODE REVIEW

File: .skpr/dev.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+ingress:
```
Comment (acbramley) [inline .skpr/dev.yml]: Were we changing this to `domains`?

Comment (nicksantamaria) [reply .skpr/dev.yml]: Maybe - ingress makes more sense in the context of the path routing example though

**Comment by nicksantamaria** [comment]
Added some issues to document the examples - https://github.com/skpr/docs/issues

File: .skpr/dev.yml (hunk 1 above)
Comment (nickschuch) [reply .skpr/dev.yml]: Agree on `ingress`

Comment (acbramley) [reply .skpr/dev.yml]: I would argue ingress doesn't mean much to people not familiar with kube

Comment (nickschuch) [reply .skpr/dev.yml]: The only reason we would not use `domains` is because `www.example.com/api` is not a domain.

Comment (nickschuch) [reply .skpr/dev.yml]: > Were we changing this to `domains`?

you used a plural ;)

**Pull request merged**